aioauth/responses.py

```diff
@@ -51,10 +51,11 @@ class TokenResponse:
     Used by :py:class:`aioauth.response_type.ResponseTypeToken`.
     """
 
-    expires_in: int
-    refresh_token_expires_in: int
     access_token: str
+    expires_in: int
+    id_token: str
```

Comment:
I think `id_token` still should be optional. For example, we don't need it in [m2m authorization](https://auth0.com/blog/using-m2m-authorization/). What do you think?